test/data/deliusClient.test.js

```diff
@@ -118,4 +118,18 @@ describe('deliusClient', () => {
       ])
     })
   })
+
+  describe('addResponsbileOfficerRole', () => {
+    test('should return data from api', () => {
+      fakeDelius.put(`/users/bobUser/roles/${config.delius.responsibleOfficerRoleId}`).reply(200, {})
+
+      return expect(deliusClient.addResponsibleOfficerRole('bobUser')).resolves.toStrictEqual({})
+    })
```

Comment:
Why does the PUT return an empty object? Why return anything?

Reply:
now not returning anything from stubbing - response body still resolves to empty object